content/blog/architect-aws-application-infra-with-pulumi-stack-references/index.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Architect AWS Application Infrastructure with Pulumi Stack References"
+title_tag: "Architect AWS Application Infrastructure with Stack References"
```

Comment:
This, for example, ends up getting rendered with a title tag of:

![image](https://user-images.githubusercontent.com/274700/70100554-ecef7980-15e6-11ea-8865-6af3405c24c6.png)